spec/features/team/show_team_spec.rb

```diff
@@ -21,4 +21,22 @@
       expect(page).to have_text another_user.first_name
     end
   end
+
+  describe 'display consecutive numbering of members' do
```

Comment:
Actually, the consecutive numbering isn't tested here, but only the table head, right? It may be nice to test that two consecutively added members have consecutive numbers.

Reply:
https://www.youtube.com/watch?v=2nsbDyjn3L4